UltraStar Play/Assets/Common/Util/StringUtils.cs

```diff
@@ -0,0 +1,7 @@
+public static class StringUtils
+{
+    public static bool IsNullOrEmpty(this string txt)
+    {
+        return string.IsNullOrEmpty(txt);
```

Comment:
I wanted the method as Extension Method. This way it can be called as `myString.IsNullOrEmpty()` instead `string.IsNullOrEmpty(myString)`. Thus, it removes the redundant `string.`.
This is also consistent with the CollectionExtensions.IsNullOrEmpty();
So you can write `if(myList.IsNullOrEmpty()) {...}` and now also `if(myString.IsNullOrEmpty()) {...}`

However, I just noticed that the class should be called `StringExtensions` not `StringUtils`

Reply:
Done